apps/slack/lambda/lib/helpers/getWorkspaceId.ts

```diff
@@ -4,13 +4,18 @@ import { NotFoundException } from '../errors';
 export async function getWorkspaceId(
   spaceId: string,
   environmentId: string,
+  host: string,
   workspaceIdFromParameters?: string
```

Comment:
nit: Shall we pass an object here. We now have 4 string arguments that are easy to mix up